dev/tests/integration/testsuite/Magento/Customer/Model/OptionsTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Customer\Model;
+
+use Magento\Config\Model\Config\Source\Nooptreq;
+use Magento\Customer\Helper\Address;
+use Magento\Framework\App\Config\MutableScopeConfigInterface;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Store\Model\ScopeInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test for \Magento\Customer\Model\Options.
+ * @magentoDbIsolation enabled
```

Comment:
We don't need interaction with a real DB to check this logic, please convert to unit test.